docs/articles/documentation/using-testcafe/common-concepts/inject-scripts-into-tested-pages.md

```diff
@@ -443,3 +443,14 @@ window.addEventListener('DOMContentLoaded', function () {
 fixture `My fixture`
     .clientScripts({ content: scriptContent });
 ```
+
+## Inject Scripts Into Iframes
+
+To inject a script into an iframe, specify the iframe URL in the [page](#provide-scripts-for-specific-pages) property.
```

Comment:
IMO this paragraph should be closer to the `Provide Scripts for Specific Pages` section because it's quite related to it

Reply:
The technique is the same, but tasks...
I would say that injecting scripts into iframes is good at the top level as ti's a separate task, not yet another way to customize page injection.